knowledge_repo/app/templates/markdown-rendered.html

```diff
@@ -50,7 +50,7 @@
                   <div class="row">
                     <div class="col-md-11">
                       <div class="post_comment">
-                        Posted by <b>{{ item.author }}</b> on {{ item.created_at_fmt }}
+                        Posted by <b>{{ item.author }}</b> on {{ item.format_created_at }}
```

Comment:
Yeah - I think longterm this is probably better, but I'll merge for now and then do a bigger sweep of switching over all properties to our own jinja filters.